packages/rainbowkit/src/components/ModalSelection/ModalSelection.tsx

```diff
@@ -88,7 +90,9 @@ export const ModalSelection = ({
           <Box alignItems="center" display="flex" flexDirection="row" gap="12">
             <AsyncImage
               background={iconBackground}
-              {...(isMouseOver ? {} : { borderColor: 'actionButtonBorder' })}
+              {...(!isMouseOver && isRainbowKitConnector
+                ? { borderColor: 'actionButtonBorder' }
+                : {})}
```

Comment:
Needed this since EIP6963 do not want to have a border color. It looks a bit ugly if i do it.